src/Http/Http.Abstractions/src/HttpContext.cs

```diff
@@ -90,7 +95,7 @@ private sealed class HttpContextDebugView(HttpContext context)
         public HttpContextFeatureDebugView Features => new HttpContextFeatureDebugView(_context.Features);
         public HttpRequest Request => _context.Request;
         public HttpResponse Response => _context.Response;
-        public Endpoint? Endpoint => _context.GetEndpoint();
+        public Endpoint? Endpoint => _context.Endpoint; // Direct access
```

Comment:
```suggestion
        public Endpoint? Endpoint => _context.Endpoint;
```
Is the comment needed?